src/configs/axios.ts

```diff
@@ -3,11 +3,10 @@ import axios from 'axios';
 import { PAGE_URL } from './path';
 
 export const API = axios.create({
-  baseURL: 'https://causw-dev.du.r.appspot.com',
-
-  // process.env.REACT_APP_ENV === 'production'
-  //   ? 'https://causw-prod.du.r.appspot.com'
-  //   : 'https://causw-dev.du.r.appspot.com',
+  baseURL:
+    process.env.NODE_ENV !== 'development'
+      ? import.meta.env.VITE_MAIN_SERVER_URL
```

Comment:
관용적으로 PROD_SERVER_URL을 사용하는게 좋을 것 같아요
```suggestion
      ? import.meta.env.VITE_PROD_SERVER_URL
```